src/main/java/xyz/holocons/mc/holoitemsrevamp/collection/CollectionManager.java

```diff
@@ -572,7 +573,7 @@ public List<Component> getLore() {
     }
 
     private static Idol buildKronii(HoloItemsRevamp plguin) {
-        return new Idol() {
+        return new Idol(new TimeLapseroniItem(plguin)) {
```

Comment:
Ah... Please add a commit to fix the typo.

Reply:
Yup, will do after school. Is there anything else to be done for this item?